### PR TITLE
docs: try to explain excludes better for #370

### PR DESCRIPTION
**Pull request by klaernie**
Hopefully this provides some better explanations for the issue seen in #370.

Reading through the section in the README I found it confusing to have the general explanation at the far bottom, when it is applicable to the understanding of all the options.

I'm very aware that if this PR is merged before #371 I'll have to rebase that PR (or vice versa).

### CODE REVIEW

**Comment by klaernie** [comment]
I was also wondering if the options should be renamed:
- `IgnoreDefaults` to `IgnoreDefaultExcludePaths`
- `Exclude` to `ExcludePaths`

These might be added in addition to the existing names, being merged in. Technically that would then allow `.ecrc` files with both `.Exclude` and `.ExcludePaths`, but would retain support for all the `.ecrc` files currently out in the wild, while still making new configs better to read.

**Review by mstruebing — APPROVED** [review]
Thank you very much, I like the explanation.

File: README.md [hunk 1]
```diff
@@ -229,7 +228,18 @@ const myTemplateString = `
...
 
-### Excluding Files
+### Excluding Paths
+
+You can exclude paths from being checked in several ways:
+- ignoring a file by documenting it inside the to-be-excluded file
+- adding a regex matching the path to the `.ecrc`
+- passing a regex matching the path as argument to `--exclude`
+
+All these excludes are used in addition to the [default excludes](#default-excludes), unless you opt out of them.
```
Comment (theoludwig) [inline README.md:238]: To make it clearer. :smile:

```suggestion
All these excludes are used in addition to the [default excludes](#default-excludes), unless you opt out of them with the `--ignore-defaults` flag.
```

**Review by theoludwig — APPROVED** [review]
Apart from the little suggestion, it looks good to me, thanks!

File: README.md (hunk 1 above)
Comment (klaernie) [reply README.md]: I fixed this a slightly different way: I created an entire new section describing the `IgnoreDefaults` option and command line switch, linking to it from all the places it is mentioned.

**Pull request merged**